pkg/apis/service/v0alpha1/register.go

```diff
@@ -0,0 +1,38 @@
+package v0alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GroupName is the group name used in this package
+const GroupName = "service.grafana.app"
```

Comment:
how is this different from `GROUP`?

Reply:
Looks like the same value. I would let @toddtreece comment. He worked on the `externalname` part and would be most aware of what is codegen here and what's authored by us.

Reply:
i think this is just a byproduct of merging the k8s pattern for registration with the grafana pattern. this one can be removed and replaced with `GROUP` where appropriate